.dockerignore

```diff
@@ -0,0 +1,6 @@
+# ignoring everything
+/*
+# except what we need
+!/bin/
+!/lib/
```

Comment:
symfony-yaml parser and php-enum are in vendor and are used when running the script, should they be included here
besides bin , lib ?

Reply:
no. it will be the role of composer to download and install them in the first step of the build.